src/app/code/Dex/PriceCalculation/Model/Configuration.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
+
+namespace Dex\PriceCalculation\Model;
+
+class Configuration
+{
+    public const XML_PATH_PRODUCT_PRICE_MULTIPLIER = 'dex/price_calculation/product_price_multiplier';
```

Comment:
Ми цілком можемо використовувати цей клас для того, щоб діставати значення конфігів, використовуючи публічні геттери, що дозволить абстрагувати логіку і винести її з класів, які не відповідальні за дану роботу. Також це зменшить звязність класів високорівневої логіки з їхніми надлишково детальними реалізаціями брання значень даних конфігів.
```suggestion
    public const XML_PATH_PRODUCT_PRICE_MULTIPLIER = 'dex/price_calculation/product_price_multiplier';
    
    public function getPriceMultiplyIndex();
```